Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -348,3 +361,20 @@ extension ApolloCodegenConfiguration {
     }
   }
 }
+
+// MARK: Helper Extensions (Internal Only)
+
+extension ApolloCodegenConfiguration.SchemaTypesFileOutput {
+  var moduleName: String {
+    switch self.dependencyAutomation {
+    case let .manuallyLinked(namespace):
```

Comment:
You could actually combine these all on to one line. The compiler doesn't care that your parameter is named differently, just that they are all strings. (You'd need to bind the value to the same `moduleName` variable` though.)